ui/src/tool-panel/location-track/dialog/location-track-edit-dialog.tsx

```diff
@@ -485,6 +493,32 @@ export const LocationTrackEditDialog: React.FC<LocationTrackDialogProps> = (
                                 />
                             }
                         />
+                        <FieldLayout
```

Comment:
Vanhaa perua mutta tämä komponenttifunkkari on aika pitkäksi venynyt jo ennen näitä ja sisennyksetkin alkaa olla suht pitkällä. Tuon voisi varmaan pilkkoa. Pilkkomisen voi tehdä myös laittamalla osia tästä rakentelusta samaan tiedoston, erilliseen funkkariin jos uusi komponentti ei tunnu tarpeelliselta